src/components/AttachResult/AttachResult.ts

```diff
@@ -194,8 +203,11 @@ export class AttachResult extends Component {
         let customData: IAnalyticsCaseAttachMeta = {
             resultUriHash: this.queryResult.raw.urihash,
             author: this.queryResult.raw.author,
-            articleID: null,
-            caseID: null
+            articleID:
+                this.options.articleIdField && this.queryResult.raw[this.options.articleIdField]
+                    ? this.queryResult.raw[this.options.articleIdField]
+                    : null,
+            caseID: this.options.caseId
```

Comment:
Is there are reason why you use a different casing here compared to the option name? I would use the same casing everywhere.

`caseID` -> `caseId`
`articleID` -> `articleId`

Reply:
je travaille sur les tests! :D
le casing cétait pour que sa fit avec les alias dans les dom attributes, sinon sa ferais `data-case-i-d`

Reply:
Pour le casing, j'irais comme ceci. C'est bien ce que tu avais en tête?

    caseId: this.options.caseId

Reply:
Je viens de voir que `articleID` et `caseID` étaient déjà définies. Laisses le casing tel qu'il est alors. J'étais sur l'impression que c'était de nouveaux attributs.